registry/slashFun.js

```diff
@@ -0,0 +1,123 @@
+// @ts-check
+const u = require('./regUtils');
+
+const roll = new u.sub()
+  .setName("roll")
+  .setDescription("Roll Dice")
+  .addIntegerOption(
+    new u.int()
+    .setName("dice")
+    .setDescription("How many dice to roll? (will roll at least 1 anyway) (max of 10000)")
+    .setRequired(false)
+    .setMinValue(1)
+    .setMaxValue(10000)
+  )
+  .addIntegerOption(
+    new u.int()
+    .setName("sides")
+    .setDescription("How many sides on the dice to roll? (defaults to 6)")
+    .setRequired(false)
+    .setMinValue(1)
+    .setMaxValue(2147483647)
```

Comment:
```suggestion
    .setMinValue(2)
    .setMaxValue(5000)
```

Reply:
I think its funni to keep the one sided die, plus there are actual one sided die... also, hows about ?214748? or perhaps round it down to 100000? that does keep it from overflowing, and rolling 10000d100000 still takes less than 50 ms in the code.

Reply:
I think an even number is fine. As for the one sided die, uhhhhh i dunno about that one. it kinda defeats the purpose of the command

Reply:
you could argue it also helps with testing a little bit to be able to roll 1...